core/src/main/java/nu/yona/server/subscriptions/service/BuddyService.java

```diff
@@ -525,4 +526,45 @@ private User getBuddyUser(BuddyDTO buddy)
 		}
 		return Optional.empty();
 	}
+
+	@Transactional
+	void broadcastBuddyInfoChangedMessageIfNeeded(User updatedUserEntity, UserDTO originalUser)
+	{
+		if (isBuddyInfoChanged(updatedUserEntity, originalUser))
+		{
+			broadcastBuddyInfoChangeMessage(updatedUserEntity, originalUser);
+		}
+	}
+
+	private String getBuddyInfoChangeMessage(User updatedUserEntity, UserDTO originalUser)
+	{
+		if (!updatedUserEntity.getNickname().equals(originalUser.getPrivateData().getNickname()))
+		{
+			return translator.getLocalizedMessage("message.buddy.changed.nickname");
```

Comment:
If we can agree to make the info message broader, I propose to go with a global message: personal information change. The user can change multiple things at once.